### DISCUSSION TITLE
Examples of customized book themes

### DISCUSSION THREAD

**Discussion by fmaussion** [executablebooks/meta#509] [Q&A] [answered]
Hi Community! 

Does anyone have examples of sphinx-book-theme sites that have been "customized" for appearance a little bit?

In my organization we have three sphinx based websites, now all using the book theme (which we love), but we'd like to give each of them a slightly different visual identity and we don't know where to start from (our CSS knowledge is basically zero...). So having examples from some repositories that did tune the book them could help us getting started.

Thanks in advance!

**Comment by andersy005** [comment, marked as answer]
@fmaussion, if you haven't done so already, take a look at the dask-sphinx-theme and the pangeo-sphinx-book-theme:



- https://github.com/pangeo-forge/pangeo-sphinx-book-theme (used by https://pangeo-forge.readthedocs.io/en/latest/)
- https://github.com/dask/dask-sphinx-theme (used by https://jobqueue.dask.org/en/latest/)

**Reply by fmaussion** [reply]
Thanks! I think I'll have no other choice than teach me some css still ;-)
